### PR TITLE
2.3.x: MYFACES-4450: Render tabindex for outputLabel

### PR DESCRIPTION
**Pull request by volosied**
https://issues.apache.org/jira/projects/MYFACES/issues/MYFACES-4450

### CODE REVIEW

**Review by melloware — APPROVED** [review]

**Comment by tandraschko** [comment]
@volosied some tests failed

**Comment by volosied** [comment]
:( Looking at the failures.

File: shared/src/main/java/org/apache/myfaces/shared/renderkit/html/CommonPropertyUtils.java [hunk 1]
```diff
@@ -531,11 +531,7 @@ public static void renderLabelPassthroughPropertiesWithoutEvents(ResponseWriter
             long commonPropertiesMarked, UIComponent component) 
     throws IOException
     {
-        if ((commonPropertiesMarked & CommonPropertyConstants.ACCESSKEY_PROP) != 0)
-        {
-            HtmlRendererUtils.renderHTMLStringAttribute(writer, component,
-                    HTML.ACCESSKEY_ATTR, HTML.ACCESSKEY_ATTR);
-        }
+        renderAccesskeyTabindexProperties(writer, commonPropertiesMarked, component);
```
Comment (volosied) [inline shared/src/main/java/org/apache/myfaces/shared/renderkit/html/CommonPropertyUtils.java:534]: ACCESSKEY_PROP is rendered within renderAccesskeyTabindexProperties, so the only addition is the tabindex

**Comment by volosied** [comment]
@tandraschko  Build is passing now, and I've updated HtmlLabelRendererTest

**Comment by tandraschko** [comment]
please also check other active branches

**Comment by volosied** [comment]
@tandraschko  added PRs for the other branches

**Pull request merged**